consensus/module.go

```diff
@@ -154,6 +162,9 @@ func (*consensusModule) Create(bus modules.Bus, options ...modules.ModuleOption)
 	m.nodeId = valAddrToIdMap[address]
 	m.nodeAddress = address
 
+	m.metadataReceived = make(chan *types.StateSyncMetadataResponse, 1000)
```

Comment:
Move `1000` into a local constant and leave a TODO that we should be it configurable